src/libmongoc/tests/test-conveniences.c

```diff
@@ -127,7 +137,33 @@ tmp_bson (const char *json, ...)
    return doc;
 }
 
+const char *
+tmp_json (const bson_t *bson)
```

Comment:
This is for ease of logging BSON as a JSON string. The caller does not need to free, so this enables statements like:

```
MONGOC_DEBUG ("document: %s", tmp_json(&bson));
```

... instead of the current way:
```
char *json = bson_as_json (&bson, NULL);
MONGOC_DEBUG ("document: %s", json);
bson_free (json);
```

Reply:
Thanks!